docs/mdbook/usage/env.md

```diff
@@ -70,3 +70,8 @@ Set `NO_COLOR=true` to disable colored output in lefthook and all subcommands th
 
 Set `CLICOLOR_FORCE=true` to force colored output in lefthook and all subcommands.
 
+### `CI`
+
+Use `CI=true` to prevent lefthook from installing hooks during the postinstall script.
```

Comment:
Could you please add that this is true only for NPM packages?

Reply:
1. It's OK to use `LEFTHOOK` var
2. I think it makes sense. Currently only presence is being checked and in case of `CI=0` it will pass. We should skip if `CI` value is `0` or `false`.
3. I think it's OK to just check `process.env.CI != '0' && process.env.CI != 'false'`

Reply:
Thanks!

I went ahead and added a local `isEnabled` helper (similar to [`isEnvEnabled()` in lefthook.go](https://github.com/evilmartians/lefthook/blob/573fd6f7341fbdabeaddfbf88edbb31ac5441448/internal/lefthook/lefthook.go#L146-L153)) for clarity/readability, but I can inline if you deem it unnecessary.

Reply:
Nice! Thank you!